bpf/lib/common.h

```diff
@@ -598,6 +598,7 @@ enum {
 						 * Not used by xfrm.
 						 */
 #define	CB_CUSTOM_CALLS		CB_CT_STATE	/* Alias, non-overlapping */
+#define	CB_LB_SELECTION_RULE	CB_CT_STATE	/* Alias, non-overlapping */
```

Comment:
Can you please add a comment here about what is stored under `CB_LB_SELECTION_RULE` and how it interacts with `LB_LOCAL_BACKEND_ONLY`?